src/I18n.js

```diff
@@ -29,9 +29,12 @@ export default class I18n extends Component {
       this.options.wait = false;
     }
 
+    const { language } = this.i18n;
```

Comment:
We only need to check the _current_ language, right?

Reply:
i18next loads more than the i18next.language -> i18next.language reflects the detected language - while i18next.languages reflects what get loaded and what will be used to lookup translations (fallbacks, whitelist,...): https://github.com/i18next/i18next/blob/master/src/i18next.js#L135

so a) check for i18next.languages[0] if that is set (hasResourceBundle) it should have loaded or if like to be very save check all i18next.languages